R/prob-brier.R

```diff
@@ -148,10 +148,6 @@ brier_ind <- function(truth, estimate, case_weights = NULL) {
 
 # When `truth` is a factor
 brier_factor <- function(truth, estimate, case_weights = NULL) {
-  if (!is.factor(truth)) {
-    rlang::abort("'truth' should be a factor.")
```

Comment:
`brier_factor()` is used internally in `brier_class()`. If `truth` isn't a factor, it will be caught way before it reaches `brier_factor()`

The dev version shows this

``` r
library(yardstick)

data("two_class_example")

brier_class(two_class_example, truth = Class2, Class1)
#> Error in `validate_factor_truth_matrix_estimate()` at yardstick/R/check-metric.R:55:2:
#> ! `truth` should be a factor, not a `numeric`.
```